stellarphot/core.py

```diff
@@ -284,7 +285,7 @@ class PhotometryData(BaseEnhancedTable):
         names as values.  This is used to automatically update the column
         names to the desired names before the validation is performed.
 
-    passband_map: dict, optional (Default: None)
+    passband_map: `stellarphot.settings.PassbandMap`, optional (Default: None)
         A dictionary containing instrumental passband names as keys and
```

Comment:
Do you want to call it a `dictionary` in the documentation?  I guess It behaves like a dictionary in many ways, but you need to create it in a very different way, am I not correct?  What happens if they just pass in a `dict`?  It should fail.  Maybe we add an Exception that reports they passed in a dict and tells them how to convert it into a proper PassbandMap object?

Reply:
Oops, that was completely an oversight on my part. I'll change the wording here and below.